exporter/tencentcloudlogserviceexporter/uploader.go

```diff
@@ -43,19 +40,8 @@ type logServiceClientImpl struct {
 }
 
 // newLogServiceClient Create Log Service client
-func newLogServiceClient(config *Config, logger *zap.Logger) (logServiceClient, error) {
-	if config == nil || config.Region == "" || config.LogSet == "" || config.Topic == "" {
-		return nil, errors.New("missing logservice params: Region, LogSet, Topic")
-	}
-
-	var credential *common.Credential
-	if config.SecretID == "" || config.SecretKey == "" {
-		credential = common.NewCredential(
-			os.Getenv("TENCENTCLOUD_SECRET_ID"),
-			os.Getenv("TENCENTCLOUD_SECRET_KEY"))
-	} else {
-		credential = common.NewCredential(config.SecretID, config.SecretKey)
-	}
+func newLogServiceClient(config *Config, logger *zap.Logger) logServiceClient {
+	credential := common.NewCredential(config.SecretID, config.SecretKey)
```

Comment:
You had a logic before, where if the config parameters are empty, you'd read the values from an env var. This PR removes that logic. Is that on purpose?

Reply:
Yes, that piece of logic is redundant. Need to enforce restrictions declared in the configuration file.